controllers/database_controller.py

```diff
@@ -46,36 +46,20 @@ def get_iac_for_collection_exercise(collection_exercise_id, social=False):
     else:
         sample_unit_type = "B"
```

Comment:
Sample unit type is always business? Shouldn't this be parametrised to include social as well?